Reference Material/harcourt_db/ipbans.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+import pyshark
+import subprocess
+import sqlite3
+from sqlite3 import Error
+
+def create_connection(db_file):
+    """ create a database connection to the SQLite database
+        specified by the db_file
+    :param db_file: database file
+    :return: Connection object or None
+    """
+    try:
+        conn = sqlite3.connect(db_file)
+        return conn
+    except Error as e:
+        print(e)
+ 
+    return None
+ 
+def read_bans(conn):
+    """
+    Query all rows in the tasks table
+    :param conn: the Connection object
+    :return:
+    """
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM ipblacklist")
+ 
+    rows = cur.fetchall()
+ 
+    for row in rows:
+        lst2.append(row)
+	
+capture = pyshark.LiveCapture(interface='eth0')
+capture.sniff(timeout=50)
+#print len(capture)
+lst1=[]
+#IP bans
+lst2=[]
+
+def blockip(ip):
+	cmd="/sbin/iptables -A INPUT -s "+ip+" -j DROP"
+	print cmd
```

Comment:
This code literally doesn't work because you're mixing prints with and without parentheses.